parallel_cluster/bootstrap/fsx-preinstall-args.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Base level functions
+echo_stderr() {
+  echo "${@}" 1>&2
+}
+
+# Source config, tells us if we're on a compute or master node
+. "/etc/parallelcluster/cfnconfig"
+
+# Globals
+FSX_SCRATCH_DIR="/fsx/scratch2"
+TIMEZONE="Australia/Melbourne"
+
+# Functions
+create_fsx_dir() {
+  : '
+  Create the fsx directory, change owner to ec2 user
+  '
+  mkdir -p "${FSX_SCRATCH_DIR}"
+}
+
+change_fsx_permissions() {
+  : '
+  Change fsx permissions s.t entire directory is owned by the ec2-user
+  '
+  chown ec2-user:ec2-user "${FSX_SCRATCH_DIR}"
+}
+
+# Runtime installations and configurations
+# Processes to complete on ALL (master and compute) nodes at startup
+
+# Security updates
```

Comment:
Not entirely sure this (and immediate lines below) belongs here... perhaps better move it to `base.yml`?

Reply:
I think yum update -y is important at the start of runtime. 
I guess the timezone could be set in the yaml?
The docker service would need to be started at runtime though right?
I guess I could do the usermod though in the yaml, and that would save that line?

The shared dir (either /fsx or /efs) is dynamic, depending on what the user sets. Will leave it out of the ami for now.

Reply:
Moved timzone and addition of docker user to base yaml